storage/rocksdb/rdb_i_s.cc

```diff
@@ -2093,6 +2228,23 @@ struct st_mysql_plugin rdb_i_s_active_compact_stats = {
     0,       /* flags */
 };
 
+struct st_mysql_plugin rdb_i_s_compact_history = {
```

Comment:
We are going to need a doc ticket to be sure this gets documented here https://www.percona.com/doc/percona-server/LATEST/myrocks/information-schema-tables.html

Reply:
Already created at https://jira.percona.com/browse/PS-7442